### DISCUSSION TITLE
GitHub Suspicious Login

### DISCUSSION THREAD

**Discussion by Storm99999** [community/community#102049] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm pretty paranoid right now, so if anyone could help, please do so;
10 minutes ago, as i was browsing github, i was randomly signed out and i checked my email, and the email "support@github.com" messaged me, to reset my password. I did. I checked my security logs and was greeted with this: 
![image](https://github.com/community/community/assets/87811650/97c30e48-2d06-4f2c-9b1f-5f4dcdf3feee)
Is this a petty dude trying to log into my account or is it a error by github?

**Comment by IceyFL** [comment]
I would either change your github to a different email and change the password or change the password to your email and your github

**Comment by queenofcorgis** [comment]
Hi there, thanks for sharing with us. This protocol is normal and is triggered in the event of an user account security concern. If you haven't already, please use our [Reinstatement Form](https://support.github.com/contact/reinstatement) to recover your account and our team will get you sorted.

**Reply by Storm99999** [reply]
i already have the account, its all sorted out

**Comment by Jaic1** [comment]
Same issue, my friend's account @zhangt2333 (a GitHub Pro user) got flagged dut to password randomized.

![image](https://github.com/community/community/assets/35210901/7fad7082-712e-41b8-abb3-5e4760a2cba7)

The password has been changed and 2FA has also been enabled.

The reinstatement request [#2570849](https://support.github.com/ticket/personal/0/2570849) has been opened but there is no any answer after 21 days.

If someone from the github staff reads this, please help us. Thank you!

**Reply by zhangt2333** [reply]
My account have been unflagged 5 minutes ago, thanks :)

**Comment by luckymanbet** [comment]
It's understandable to feel paranoid in such situations, but it's important to remain calm and take appropriate steps to address the issue. Based on the information you provided, it's possible that someone may have attempted to access your GitHub account. Here's what you can do:

1. Review the Email: Verify the authenticity of the email you received from "support@github.com." Make sure it's not a phishing attempt by checking the sender's email address, grammar, and any suspicious links included in the email. Legitimate emails from GitHub typically contain information about the password reset request and instructions on how to proceed securely.

2. Check the Security Logs: Since you mentioned checking your security logs, review them carefully to see if there are any other suspicious activities or login attempts from unfamiliar IP addresses or locations. This can help you determine if the password reset request was initiated by someone other than yourself.

3. Change Your Password Again: If you suspect unauthorized access to your account, it's crucial to change your password immediately to prevent further unauthorized activity. Choose a strong, unique password that is not easily guessable and enable two-factor authentication for added security.

4. Contact GitHub Support: If you're unsure about the legitimacy of the password reset request or if you believe your account has been compromised, reach out to GitHub support for assistance. They can provide guidance on securing your account and investigate any suspicious activities.

5. Monitor Your Account: After taking these steps, continue to monitor your GitHub account for any unusual activities or signs of unauthorized access. Regularly review your security settings and consider enabling notifications for account login attempts or changes to your account settings.

It's important to remain vigilant and take proactive measures to protect your online accounts from unauthorized access. If you have any concerns about the security of your GitHub account or need further assistance, don't hesitate to reach out to GitHub's support team for help.

**Reply by queenofcorgis** [reply]
@austinleepro 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by ghost** [comment]
My account @nova1751 also have the same problem.It all happened just because I synced one of my forked repo from its upstream in the repo page. It may related to github desktop pushing too much commits at once. Someone also have the same issue as mine. My ticket is [#2701654](https://support.github.com/ticket/2701654).
If someone from the github staff see this, I would really appreciate it if you could provied some help. Thank you!

**Reply by nova1751** [reply]
My account problem is resolved, thanks for the support.

**Comment by ghost** [comment]
Just got this issue half an hour ago, still stuck and about to factory reset my Macbook. Help!

**Comment by bentwnghk** [comment]
I am experiencing this same issue with my account @bentwnghk. It got flagged due to password randomized, and it has become limited in many different ways.
I submitted a reinstatement request already but so far no response has been heard. If someone from the Github staff reads this, please help. Thank you!

**Reply by github-staff** [reply]
@bentwnghk Thank you for your patience. Our Platform Health team has investigated this issue and found the root cause to be an error in how malicious behavior was flagged, causing the suspension of several accounts. This error has been resolved and the affected accounts have been reinstated.
We apologize for the inconvenience this caused. If your account is still currently suspended, please reach out to Support for further investigation.

**Comment by OneOfTheMicahs** [comment]
> @bentwnghk Thank you for your patience. Our Platform Health team has investigated this issue and found the root cause to be an error in how malicious behavior was flagged, causing the suspension of several accounts. This error has been resolved and the affected accounts have been reinstated. We apologize for the inconvenience this caused. If your account is still currently suspended, please reach out to Support for further investigation.

I'm still flagged and unable to contribute on any projects. I've reached out to support but haven't heard anything.

**Comment by prascle** [comment]
Hello, I am also experiencing a similar problem with my @prascle account. I still don't know why my account is no longer visible, and my reinstatement request [2679356](https://support.github.com/ticket/2679356) is still unanswered after more than 3 weeks. If anyone from the GitHub team is reading this, please explain what caused this problem and fix it if possible! 
Thank you!

**Comment by CoderOMaster** [comment]
hello i am experiencing same issue and i noticed some unwanted activites from 3 different locations..please help me recover the account and if possible do look into details of who tried to access it please @github-staff
